src/NUnitFramework/Common.props

```diff
@@ -0,0 +1,21 @@
+<Project>
+
+  <PropertyGroup>
+    <!-- netcoreapp1.1 tests currently pull the netstandard1.6 framework, so they should not have PARALLEL defined yet. -->
+    <DefineConstants Condition="'$(TargetFramework)' != 'netstandard1.6' and '$(TargetFramework)' != 'netcoreapp1.1'">$(DefineConstants);PARALLEL</DefineConstants>
+    <DefineConstants Condition="'$(TargetFramework)' != 'net20' and '$(TargetFramework)' != 'net35'">$(DefineConstants);ASYNC</DefineConstants>
+
+    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <SignAssembly>true</SignAssembly>
+    <AssemblyOriginatorKeyFile>..\..\nunit.snk</AssemblyOriginatorKeyFile>
+
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <OutputPath>..\..\..\bin\$(Configuration)\</OutputPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <Compile Include="..\..\CommonAssemblyInfo.cs" Link="Properties\CommonAssemblyInfo.cs" />
+    <Compile Include="..\FrameworkVersion.cs" Link="Properties\FrameworkVersion.cs" />
```

Comment:
Should we also have a link to `nunit.sn` as before?

Reply:
I don't see any reason; it's just visual clutter in each project.

Reply:
Ok 👍